### PR TITLE
remove kyma-specifics from the prometheus example

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**
As the monitoring component got removed (https://github.com/kyma-project/kyma/issues/16306), the prometheus example can get simplified by removing the kyma specifics

Changes proposed in this pull request:

- remove namespace exclusion
- remove custom port for node-exporter
- remove disablement of kublet-service
- improved descriptions
- introduced an overall overview page taken from https://github.com/kyma-project/kyma/pull/18420

**Related issue(s)**

### CODE REVIEW

File: prometheus/README.md [hunk 1]
```diff
@@ -1,140 +1,23 @@
-# Install a custom kube-prometheus-stack in Kyma
+# Monitoring in Kyma using a custom kube-prometheus-stack
 
 ## Overview
 
-The Kyma monitoring stack brings limited configuration options in contrast to the upstream [`kube-prometheus-stack`](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack) chart. Modifications might be reset at the next upgrade cycle.
+The Kyma telemetry module provides collection and integration into observability backends based on OpenTelemetry. OpenTelemetry is the new vendor-neutral player in the cloud-native observability domain and has a growing adoption. However, it still lacks many features and sometimes the typical `kube-prometheus-stack` based on Prometheus, Grafana and the surrounding helper tools is more appropriate.
```
Comment (mmitoraj) [inline prometheus/README.md:5]:
```suggestion
The Kyma Telemetry module provides collection and integration into observability backends based on OpenTelemetry. OpenTelemetry is the new vendor-neutral player in the cloud-native observability domain and has a growing adoption. However, it still lacks many features and sometimes the typical `kube-prometheus-stack` based on Prometheus, Grafana, and the surrounding helper tools is more appropriate.
```

File: prometheus/README.md [hunk 2]
```diff
@@ -1,140 +1,23 @@
...
 
-### Preparation
-1. If you cluster was installed manually using the Kyma CLI, you must assure that the Kyma monitoring stack running in your cluster is limited to detection of Kubernetes resources only in the `kyma-system` Namespace. To rule out that there are any side effects with the additional custom stack, run:
-    ```bash
-    kyma deploy --component monitoring --value monitoring.prometheusOperator.namespaces.releaseNamespace=true
-    ```
+1. [**Deploy a custom Prometheus stack**](./prometheus.md), in which you deploy the [kube-prometheus-stack](https://github.com/prometheus-operator/kube-prometheus) from the upstream Helm chart.
 
-1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+2. [**Observe application metrics**](./monitoring-custom-metrics/README.md), in which you redirect the `cpu_temperature_celsius` metric to the localhost and the Prometheus UI. You later observe how the metric value changes in the predefined 10 seconds interval in which Prometheus scrapes the metric values from the service's `/metrics` endpoint.
```
Comment (mmitoraj) [inline prometheus/README.md:19]:
```suggestion
2. [**Observe application metrics**](./monitoring-custom-metrics/README.md), in which you redirect the `cpu_temperature_celsius` metric to the localhost and the Prometheus UI. You later observe how the metric value changes in the predefined 10-second interval in which Prometheus scrapes the metric values from the service's `/metrics` endpoint.
```

File: prometheus/values.yaml [hunk 3]
```diff
@@ -1,12 +1,7 @@
-####### This block is required to run in parallel with a Kyma monitoring stack
-prometheusOperator:
-  # exclude the kyma-system Namespace from watching to avoid conflict with the Kyma monitoring stack
-  denyNamespaces:
-    - kyma-system
-  # re-use the kublet service definition of the Kyma monitoring stack.
-  kubeletService:
-    enabled: false
 
+####### This block configures resource limits for the tooling and disables scaping of them via annotation
```
Comment (mmitoraj) [inline prometheus/values.yaml:2]:
```suggestion
####### This block configures resource limits for the tooling and disables scaping of them via annotations
```

File: prometheus/values.yaml [hunk 4]
```diff
@@ -37,45 +30,49 @@ kube-state-metrics:
       cpu: "500m"
       memory: "2Gi"
 
-####### This block disables control plane components which are not reachable from within the Gardener data plane
+####### This block disabled not needed features
```
Comment (mmitoraj) [inline prometheus/values.yaml:33]:
```suggestion
####### This block disables not needed features
```
"Disabled" or "disables"?

**Review by mmitoraj — APPROVED** [review]

**Review by chrkl — APPROVED** [review]

**Pull request merged**